Ninja-star/NinjaStar/GamePlayScene.swift

```diff
@@ -0,0 +1,122 @@
+//
+//  GameScene.swift
+//  NinjaStar
+//
+//  Created by Nissan Adid on 3/03/18.
+//  Copyright © 2018 Nissan Adid. All rights reserved.
+//
+
+import SpriteKit
+import GameplayKit
+
+class GamePlayScene: SKScene {
+   
+   private var label : SKLabelNode?
+   private var spinnyNode : SKShapeNode?
+   
+   //create the ninja
+   private let ninja = Ninja()
+   
+   //background
+   private let background = SKSpriteNode(imageNamed: "GameBackground")
+   
+   //Ground
+   private let ground = SKSpriteNode()
+   
+   override func didMove(to view: SKView) {
+      super.didMove(to: view)
+      
+      
+      //set the ninja  position in the scene
+      ninja.position = CGPoint(x: 100, y: 100)
+      
+      //set-up the background position and size to fill the window
+      backgroundColor = SKColor.black
+      background.zPosition = -1
+      background.size.width = UIScreen.main.bounds.size.width
+      background.size.height = UIScreen.main.bounds.size.height
+      addChild(background)
+      
+      //Set-up the ground so that the player does not fall through the ground. Also place it on the same zposition as the player
+      ground.color = UIColor.clear
+      ground.zPosition = 1
+      let sizeOfGround = CGSize(width: UIScreen.main.bounds.size.width, height: 1)
+      ground.size.width = sizeOfGround.width
+      ground.size.height = sizeOfGround.height
+      ground.position.y = -100
+      ground.physicsBody = SKPhysicsBody(rectangleOf: sizeOfGround)
+      ground.physicsBody?.affectedByGravity = false
+      ground.physicsBody?.isDynamic = false
+      addChild(ground)
+      //add ninja as a child of the scene
+      addChild(ninja)
+      ninja.zPosition = 1
```

Comment:
Looks like the ground and the ninja are on the same index, this could cause rendering issues.